lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -42,7 +43,9 @@ def precompile_assets
 
       def precompile_sti_loader
         Dir.chdir(miq_dir) do
-          shell_cmd("bundle exec rake evm:compile_sti_loader")
+          shell_cmd("BUNDLER_GROUPS=manageiq_default,ui_dependencies,graphql_api bundle exec rake evm:compile_sti_loader")
```

Comment:
is it possible to pass this as an env variable instead of as command params?

not sure how much work it would be for shell_cmd to support passing that across.

Reply:
I started looking at that actually but 1. I'd have to refactor shell_cmd and 2. we pass env vars in this way elsewhere in this file (specifically RAILS_ENV=production above)

Reply:
Would like to do this as a follow-up though, don't want to throw too much else into this PR

Reply:
yea was looking at https://apidock.com/ruby/Kernel/system

sample given was

```
# set FOO as BAR and unset BAZ.
pid = spawn({"FOO"=>"BAR", "BAZ"=>nil}, command)
```

I'm pretty sure the `ENV` passed to `spawn` / `system` is cumulative but we probably want to double check.

I agree that just copying current style and not making tons of changes is the smartest plan.